worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -909,13 +901,26 @@ class PushDataHandler extends BaseMessageHandler with Logging {
         callback,
         wrappedCallback)) return
 
+    // During worker shutdown, worker will return HARD_SPLIT for all existed partition.
```

Comment:
Better to rename `handleRpcRequestCore` to `handleMapPartitionRpcRequestCore` since it's only used for MapPartition.